envs/monkey_zoo/packer/smb.pkr.hcl

```diff
@@ -0,0 +1,77 @@
+packer {
+  required_plugins {
+    googlecompute = {
+      source  = "github.com/hashicorp/googlecompute"
+      version = "~> 1"
+    }
+    ansible = {
+      source  = "github.com/hashicorp/ansible"
+      version = "~> 1"
+    }
+  }
+}
+
+variable "project_id" {
+    type = string
+}
+variable "zone" {
+    type = string
+    default = "europe-west3-a"
+}
+variable "machine_type" {
+    type = string
+    default = "e2-standard-4"
+}
+variable "source_image" {
+    type = string
+    default = "windows-server-2016-dc-v20211216"
+}
+variable "account_file" {
+    type = string
+}
+variable "packer_username" {
+    type = string
+    default = "packer_user"
+}
+variable "packer_user_password" {
+    type = string
+    default = "Passw0rd"
+}
+
+
+
+source "googlecompute" "mimikatz-15" {
```

Comment:
If packer scripts are **also documentation about the machine**, we should add a description that explains the reason why we need this machine, and what's it used for.

Reply:
Packer scripts are configuration about machines, not documentation. For documentation we have zoo_network.